app/assets/stylesheets/_honeycrisp-compact.scss

```diff
@@ -324,10 +324,14 @@
     justify-content: space-between;
   }
 
-  &.flex-end {
+  &.justify-flex-end {
```

Comment:
This `justify-flex-end` class appears unused. Consider deleting it.

When I search the code for templates that use it, I find only `home.html.erb` that you have changed below.

```
$ git grep flex-end -- '*.html.erb'
app/views/ctc/ctc_pages/home.html.erb:      <div class="grid-flex flex-end">
```